homeassistant/components/pyload/sensor.py

```diff
@@ -127,96 +117,50 @@ async def async_setup_platform(
 
 async def async_setup_entry(
     hass: HomeAssistant,
-    entry: PyLoadConfigEntry,
+    entry: ConfigEntry,
```

Comment:
Que?

Reply:
that caused somehow circle imports

Reply:
Where did you import it from? It should work when you import from init

Reply:
ok, now it works, no idea what i did to get that error 😬